code/__HELPERS/mobs.dm

```diff
@@ -205,7 +205,7 @@
 		"balls_efficiency"	= CUM_EFFICIENCY,
 		"has_breasts" 		= FALSE,
 		"breasts_color" 	= pick("FFFFFF","7F7F7F", "7FFF7F", "7F7FFF", "FF7F7F", "7FFFFF", "FF7FFF", "FFFF7F"),
-		"breasts_size" 		= pick(CONFIG_GET(keyed_list/breasts_cups_prefs)),
+		"breasts_size" 		= pick(GLOB.breast_values),
```

Comment:
there should be a way to configure what breast sizes are available for players to pick tho